pages/dkp/kommander/2.1/projects/applications/catalog-applications/custom-resources-workspace-catalog/zookeeper/index.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: layout.pug
+navigationTitle: ZooKeeper
+title: ZooKeeper
+menuWeight: 5
+excerpt: Deploying ZooKeeper in a project
+---
+
+## Getting Started
```

Comment:
```suggestion
## Getting started
```

Reply:
fixed in cfa2be8cb5c4e4a2631c3ad1c19923ea6495cc86